java/arcs/core/tools/BUILD

```diff
@@ -0,0 +1,21 @@
+load("@rules_java//java:defs.bzl", "java_binary")
+load("//tools/build_defs/kotlin:rules.bzl", "kt_jvm_library")
+
+package(default_visibility = ["//visibility:public"])
+
+kt_jvm_library(
```

Comment:
If `arcs/core/tools` is going to consist of command-line JVM-based tooling, it might make sense to create a README.md in this directory to explain that.

Reply:
Good idea.